app.vue

```diff
@@ -17,4 +17,19 @@ import { ITheme } from './utils/theme'
 AppSetup()
 const theme = useState<ITheme>('theme.current')
 const locale = useState<string>('locale.setting')
+const app = useAppConfig()
+
+useHead({
```

Comment:
I thought to keep putting meta like this in nuxt.config.ts. because I think it will make the code cleaner, and other people can do this configuration easily in config not in .vue file.

do you have any further reasons for this?

Reply:
I don't think head data should be placed in `nuxt.config.ts`, this file should config build-related information. Moreover, head data will most likely not be changed, so may wish to put it in app.vue